deckhouse-controller/go-build.sh

```diff
@@ -17,11 +17,12 @@
 set -e
 
 deckhouseVer=${D8_VERSION:-"dev"}
+defaultKubernetesVer=${DEFAULT_KUBERNETES_VERSION:-"1.30"}
```

Comment:
maybe better fail with error if `DEFAULT_KUBERNETES_VERSION` is empty?
let user set version manually

Reply:
done